cloudbuild-pr.yaml

```diff
@@ -28,7 +28,6 @@ steps:
     - 'ESRA_URL=$_ESRA_URL'
     - 'KEYCLOAK_CONFIG_PATH=$_KEYCLOAK_CONFIG_PATH'
     - 'NAME_REQUEST_URL=$_NAME_REQUEST_URL'
-    - 'ONESTOP_URL=$_ONESTOP_URL'
```

Comment:
I don't know if/where this in 1Password, nor whether any other project uses this key, but THIS project will no longer need it once this change has gone to Prod.

^^ Patrick